### PR TITLE
Case 21642 - verification dialog when user hits return in goto search

### PR DESCRIPTION
**Pull request by roxanneskelly**
text entry+return should throw up verification dialog for first time to
non-optimized domain

### CODE REVIEW

**Comment by roxanneskelly** [comment]
https://highfidelity.manuscript.com/f/cases/21642/Stand-alone-Tags-Goto-Menu-text-entry-return-should-throw-up-verification-dialog-for-first-time-to-non-optimized-domain

File: interface/resources/qml/hifi/Feed.qml [hunk 1]
```diff
@@ -189,4 +189,13 @@ Column {
             }
         }
     }
+    function isStandalone(address) {
+    
+        for (var i=0; i < suggestions.count; i++) {
+            if (suggestions.get(i).place_name.toLowerCase() === address.toLowerCase()) {
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/Feed.qml]: Could probably keep address.toLowerCase() outside the loop.

Comment (roxanneskelly) [reply interface/resources/qml/hifi/Feed.qml]: good point, will fix

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18017/HighFidelity-Beta-PR15145-b3f5d76.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18017/HighFidelity-Quest-Beta-PR15145-b3f5d76.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18017/HighFidelity-Beta-PR15145-b3f5d76.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18017/HighFidelity-Beta-Interface-PR15145-b3f5d76.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18017/HighFidelity-Beta-PR15145-b3f5d76.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18017/HighFidelity-Beta-Interface-PR15145-b3f5d76.exe

**Pull request merged**